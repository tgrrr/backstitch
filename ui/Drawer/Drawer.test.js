import React from 'react'
import renderer from 'react-test-renderer'
import { Drawer } from '.'

it('Drawer: default', () => {
    const component = renderer.create(
        <Drawer>
            <>
                Drawer
            </>
        </Drawer>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
