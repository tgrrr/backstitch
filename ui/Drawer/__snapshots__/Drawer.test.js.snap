// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drawer: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Drawer "
    data-testid="Drawer"
  />
</div>
`;
